XForm/XForm/Functions/IsNull.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using XForm.Data;
+
+namespace XForm.Functions
+{
+    internal class IsNullBuilder : IFunctionBuilder
+    {
+        public string Name => "IsNull";
```

Comment:
I usually put Name, Usage, ReturnType together on consecutive lines. Just short and consistent.